### PR TITLE
intro-to-web-development: content without diagrams

### PR DESCRIPTION
**Pull request by tildeee**
At the top: Instructor file for the topic
1. Web Applications and Layers
2. Intro to Clients Servers
3. Sequence Diagrams
4. Activity

### CODE REVIEW

File: intro-to-web-development/activity-layers.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+When possible, these pairs of 2-3 people should compare and discuss their diagrams with another pair. Follow classroom instructions.
+
+### Preparation
+
+Groups will be making diagrams together during this time. Folks should prepare to collaborate, so preparing the following supplies is helpful:
+
+- Physical supplies, such as paper, markers, pens, scissors, glue, etc.
+- Screen sharing tools
+- Diagram-making tools
+
```
Comment (tildeee) [inline intro-to-web-development/activity-layers.md:25]:
```suggestion
- Diagram-making tools and digital drawing tools, such as canva.com, sketch.io, lucidchart.com, etc.

```

@tgoslee Is something like this what you had in mind?

Comment (tgoslee) [reply intro-to-web-development/activity-layers.md:25]: yep!

**Review by tgoslee — APPROVED** [review]

File: intro-to-web-development/sequence-diagrams.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+In sequence diagrams, each vertical column represents an object, actor, or process.
+
+Because this curriculum encourages using sequence diagrams in the informal, abstract ways, examples of different objects, actors, or processes we may see are:
+
+1. User, front-end, back-end API, database
+1. End-user, client, server
+
+## Horizontal Lines Are Messages Passed Inbetween
+
+In sequence diagrams, each horizontal line is usually an arrow that represents a message being sent. **Message** is a generic term to mean one discrete piece of communication. Messages can likely be an HTTP request or HTTP response. However, messages could also be a function call, or an abstract message.
```
Comment (jbieniosek) [inline intro-to-web-development/sequence-diagrams.md:51]:
```suggestion
In sequence diagrams, each horizontal line is usually an arrow that represents a message being sent. **Message** is a generic term to mean one discrete piece of communication. Messages are often an HTTP request or HTTP response. However, messages can also be a function call, or an abstract message.
```

File: intro-to-web-development/sequence-diagrams.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+* id: Coou7j
+* title: Sequence Diagrams
+##### !question
+
+Given this diagram, order these following options in sequential order. The first message sent should be listed first, and the last message sent should be listed last.
+
+##### !end-question
+##### !answer
+
+1. User
```
Comment (jbieniosek) [inline intro-to-web-development/sequence-diagrams.md:92]: 'User' doesn't seem to match the other options - all of the other options are an action by an entity.  User is just an entity.  Suggest changing to "User clicks on [something]"

Comment (tildeee) [reply intro-to-web-development/sequence-diagrams.md:92]: Fantastic call!! Thanks.

I'll update it now with some "dummy" text, although I wonder if I'll be able to get a sequence diagram to actually base this off (Now when I look at these two CFU questions again, they make nooooo sense hahahaha)

```suggestion
1. User clicks on the home button of a web page
```

**Pull request merged**